Cargo.toml

```diff
@@ -56,6 +56,9 @@ tokio-stream = { version = "0.1.12", features = ["net"] }
 totp-lite = "2.0.0"
 url = "2.3.1"
 uuid = { version = "1.3.0", features = ["v4"] }
+webauthn-authenticator-rs = { version="0.5.0-dev", git = "https://github.com/kanidm/webauthn-rs.git", url  = "./webauthn-authenticator-rs/", features = ["usb"], optional = true }
+webauthn-rs-proto = { version="0.5.0-dev", git = "https://github.com/kanidm/webauthn-rs.git", url  = "./webauthn-rs-protocol/", optional = true }
+webauthn-rs = { version="0.5.0-dev", git = "https://github.com/kanidm/webauthn-rs.git", optional = true }
```

Comment:
Is there a big difference between this version and the one currently released on crates.io (`0.4.8`)?